backend/src/core/features/game-identifier/game.identifier.js

```diff
@@ -148,39 +145,15 @@ export class GameIdentifier {
 
     const steamApps = await this.#steamAppsRepository.getSteamAppsById(games.ids);
 
-    const htmlDetailsPages = await this.#getSteamAppsHtmlDetailsPages(
-      steamApps.content,
-      source,
-    );
+    const steamApiApps = await this.#getSteamAppsViaSteamApi(steamApps.content);
 
-    steamApps.recordAttemptsViaSource(htmlDetailsPages, source);
+    steamApps.recordAttemptsViaSteamApi(steamApiApps);
```

Comment:
You can decouple the naming of the data model methods from the raw data source. By now you should be using the `steamAppsRaw` data structure so it will be enough to name the method: `recordAttempt`